app.py

```diff
@@ -21,6 +21,8 @@ class CookieNames:
 
 app = flask.Flask(__name__)
 
+app.config['SECRET_KEY'] = "SUPERSECIRTKEY"
```

Comment:
Didn't we decide to move this to `config_private.json`?

Reply:
Yes we did however it was not working for me so I just added it there for ease to work on the project